src/strings.cpp

```diff
@@ -676,9 +676,10 @@ struct UnitsLong {
 
 /** Unit conversions for velocity. */
 static const Units _units_velocity[] = {
-	{ {   1,  0}, STR_UNITS_VELOCITY_IMPERIAL },
-	{ { 103,  6}, STR_UNITS_VELOCITY_METRIC   },
-	{ {1831, 12}, STR_UNITS_VELOCITY_SI       },
+	{ {    1,  0}, STR_UNITS_VELOCITY_IMPERIAL     },
+	{ {  103,  6}, STR_UNITS_VELOCITY_METRIC       },
+	{ { 1831, 12}, STR_UNITS_VELOCITY_SI           },
+	{ {37900, 16}, STR_UNITS_VELOCITY_PLAYORIENTED },
```

Comment:
I chose the magic numbers here based on the tile length information on the wiki: [A tile is, for vehicle speed purposes 664.(216) km-ish, 668 km or 415 miles long](https://wiki.openttd.org/Game_mechanics#Vehicle_speeds).

A decitile a day = 668 km / 24 hours = 0.57820743 mph, equal to `37900 / (1<<16)` to within 0.02%.

I confirmed this by testing. (Start a new game, build a straight and level track section of known length, and see how many in-game days a train going max speed needs to pass that section.)

Reply:
I changed the constant to 37893 as that's even more accurate.  (0.57820743 * 65536 == 37893.40213248)

Reply:
Updated the constant again to 37888 after doing the math again with the 664.(216) km-ish value rather than the 668 km value.